### ISSUE TITLE
Explicitly add training arguments to all batch_norm layers in vision/beta/modeling

### ISSUE THREAD

**Issue by jrabary** [tensorflow/models#9538]
To reproduce state of the art result for some model such as DeepLab V3, several pre-training and fine-tuning steps are required. In the fine-tuning step batch norm layers should be frozen (as stated in the paper and here https://keras.io/guides/transfer_learning/#finetuning)

To do so, those layers should be call with `training=False` but all the calls in every blocks and networks defined in `official/vision/beta` don't proxy the `training` argument to the `normalisation` sublayers. For instance . https://github.com/tensorflow/models/blob/6d451a7979d670c1a6a9f995c189852d49e6254d/official/vision/beta/modeling/layers/nn_blocks.py#L58

So, I'm wondering wether the current implementation will work as expected anyway or we need to add these explicit call to `training=False`

**Comment by saberkun** [comment]
Keras framework itself will populate 'training' argument inside all layer __call__. As long as the keras model is called with training=True, it will populate to all inner layers.
It is convenient but hidden behavior. Thank you for asking the question. I believe many users/researchers have the same question. https://github.com/tensorflow/tensorflow/blob/v2.3.1/tensorflow/python/keras/engine/base_layer.py#L945

**Comment by saberkun** [comment]
Sorry for missing the feature request part. 
@arashwan

**Comment by arashwan** [comment]
Thanks Jaonary for your question. The vision beta directory is still under development. Freezing bn layers during training is indeed helpful in some cases. We plan to support this feature soon.
